utils/archiver/scenario.md

```diff
@@ -0,0 +1,48 @@
+# Running Archiver scenario
+
+## Gather DB statistics
+
+Run all SQL statements defined in [DB statistics which should be run before the archiver is started](db_scripts.md#db-statistics-which-should-be-run-before-the-archiver-is-started) section.
+
+## Run the archiver with disabled deletion
+
+Set the following environment variables:
+1. `APP_PERFORM_DELETION` set to `false`.
+2. `APP_DRY_RUN`  set to `false`.
+
+Run the archiver (`./apply.sh`).
```

Comment:
These (lines 9 and 13) seem to be subsequent steps, so I suggest numbering them (while the variables can be turned into bullet points).